pkg/webreg/webreg.go

```diff
@@ -611,37 +611,85 @@ Example:
 
 <h3 id="image">How do I use an image from another repo in my repo’s tests?</h3>
 <p>
-All images built for the release targeted by the job (ex:
-<code>release-4.2</code>) are available under <code>stable:tag-name</code>
-(example: <code>stable:installer</code> or <code>stable:cli</code>). There
-are 2 ways of using these images: creating a registry reference that can be
-used in your <code>ci-operator</code> config or adding a literal test step to
-the test in your <code>ci-operator</code> config. Images built by ci-operator
-for your repo are also available under <code>pipeline:tag-name</code>. Here
-are 2 examples of adding a very simple step that runs an oc command on the
-cluster created by <code>ipi-install</code> using the <code>stable:cli</code>
-image:
+In order to use an image from one repository in the tests of another, it is necessary
+to first publish the image from the producer repository and import it in the consumer
+repository. Generally, a central <code>ImageStream</code> is used for continuous
+integration; a repository opts into using an integration stream with the <code>tag_specification</code>
+field in the <code>ci-operator</code> configuration and opts into publishing to the
+stream with the <code>promotion</code> field.
 </p>
 
-Both examples assume we are starting off with this ci-operator config:
-{{ yamlSyntax (index . "imageExampleBaseConfig") }}
+<h4 id="image-publication">Publishing an Image For Reuse</h3>
+<p>
+When configuring <code>ci-operator</code> for a repository, the <code>promotion</code>
+stanza declares which container images are published and defines the integration stream where
+they will be available. By default, all container images declared in the <code>images</code>
+block of a configuration are published. Promotion can be configured to exclude any of those
```

Comment:
promotion declares the imagestream where things are promoted to.  where are things promoted to when they are promoted by default?  can it differ from where the promotion stanza promotes to?

Reply:
your example is still missing an "exclude" example.